src/query/functions-v2/src/scalars/comparison.rs

```diff
@@ -0,0 +1,324 @@
+                                "gte",
+                                FunctionProperty::default(),
+                                |_, _| None,
+                                |lhs, rhs, _| lhs >= rhs,
+                            );
+                        registry
+                            .register_2_arg::<NumberType<NUM_TYPE>, NumberType<NUM_TYPE>, BooleanType, _, _>(
+                                "lt",
+                                FunctionProperty::default(),
+                                |_, _| None,
+                                |lhs, rhs, _| lhs < rhs,
+                            );
+                        registry
+                            .register_2_arg::<NumberType<NUM_TYPE>, NumberType<NUM_TYPE>, BooleanType, _, _>(
+                                "lte",
+                                FunctionProperty::default(),
+                                |_, _| None,
+                                |lhs, rhs, _| lhs <= rhs,
+                            );
+                    }
+                });
+            }
+            _ => todo!(),
+        });
+    }
+
+    // TODO: wait @b41sh complete jsonb cmp, and need support
+    // `VariantType like VariantType`
+    // `VariantType regexp VariantType`
+    // registry.register_2_arg::<VariantType, VariantType, BooleanType, _, _>(
+    //     "eq",
+    //     FunctionProperty::default(),
+    //     |_lhs, _rhs| None,
+    //     |lhs, rhs, _| lhs.cmp(rhs) == Ordering::Equal,
+    // );
+
+    // registry.register_2_arg::<VariantType, VariantType, BooleanType, _, _>(
+    //     "gt",
+    //     FunctionProperty::default(),
+    //     |_lhs, _rhs| None,
+    //     |lhs, rhs, _| lhs.cmp(rhs) == Ordering::Greater,
+    // );
+
+    // registry.register_2_arg::<VariantType, VariantType, BooleanType, _, _>(
+    //     "lt",
+    //     FunctionProperty::default(),
+    //     |_lhs, _rhs| None,
+    //     |lhs, rhs, _| lhs.cmp(rhs) == Ordering::Less,
+    // );
+
+    // registry.register_2_arg::<VariantType, VariantType, BooleanType, _, _>(
+    //     "gte",
+    //     FunctionProperty::default(),
+    //     |_lhs, _rhs| None,
+    //     |lhs, rhs, _| {
+    //         let res = lhs.cmp(rhs);
+    //         res == Ordering::Equal || res == Ordering::Greater
+    //     },
+    // );
+
+    // registry.register_2_arg::<VariantType, VariantType, BooleanType, _, _>(
+    //     "lte",
+    //     FunctionProperty::default(),
+    //     |_lhs, _rhs| None,
+    //     |lhs, rhs, _| {
+    //         let res = lhs.cmp(rhs);
+    //         res == Ordering::Equal || res == Ordering::Less
+    //     },
+    // );
+
+    // registry.register_2_arg::<VariantType, VariantType, BooleanType, _, _>(
+    //     "noteq",
+    //     FunctionProperty::default(),
+    //     |_lhs, _rhs| None,
+    //     |lhs, rhs, _| lhs.cmp(rhs) != Ordering::Equal,
+    // );
+
+    registry.register_2_arg::<StringType, StringType, BooleanType, _, _>(
```

Comment:
do we need to consider using `register_passthrough_nullable_2_arg` for nullable data types?

Reply:
Yes, I try to modify this. But it occurs other err, needs to fix it next.

Reply:
sorry, `register_2_arg` is actually a wrapper of `register_passthrough_nullable_2_arg`. Don't need to modify it.